toolchain-windows.cmake

```diff
@@ -0,0 +1,73 @@
+# ########################################################################
+# Copyright (C) 2022-2023 Advanced Micro Devices, Inc.
```

Comment:
TheRock is ROCm's official windows build. If hipsdk needs a custom toolchain, then it is local to that, not global for Windows